geomstats/geometry/general_linear.py

```diff
@@ -198,50 +198,9 @@ class SquareMatrices(MatrixLieAlgebra):
         Integer representing the shape of the matrices: n x n.
     """
 
-    def __init__(self, n, equip=True):
+    def __init__(self, n, equip=True, equip_maximally=True):
```

Comment:
So `equip` refers to equip with metrics and equip_maximally to equip with structures?

maybe use `equip_with_metric` and `equip_with_structures` then?